configs/openimages/README.md

```diff
@@ -1,6 +1,9 @@
 # Open Images Dataset
-<!-- [DATASET] -->
 
+> [Open Images Dataset](https://arxiv.org/abs/1811.00982)
+
+<!-- [DATASET] -->
+<!-- [ALGORITHM] -->
```

Comment:
will get error when using pre-commit
```
check algorithm readme...................................................Failed
- hook id: check-algo-readme
- exit code: 1

Failed to find "<!-- [ALGORITHM] -->" flag from readme, please check configs/openimages/README.md again.
```

Reply:
Only algorithm class has Collections in metafile.

Reply:
Do we need to delete <!-- [ALGORITHM] --> here?